### PR TITLE
[batch] Remove batch worker ssh key

### PR DESCRIPTION
**Pull request by daniel-goldstein**
This just seems like a liability and was always awkward because we don't have a good place to keep common keys like this.

### CODE REVIEW

File: infra/azure/modules/batch/main.tf [hunk 1]
```diff
@@ -56,16 +56,6 @@ resource "azurerm_shared_image" "batch_worker_22_04" {
   }
 }
 
-resource "kubernetes_secret" "batch_worker_ssh_public_key" {
-  metadata {
-    name = "batch-worker-ssh-public-key"
-  }
-
-  data = {
```
Comment (jigold) [inline infra/azure/modules/batch/main.tf:64]: Did we have to generate this key file ourselves? Where were those instructions / commands if yes?

**Review by jigold — CHANGES_REQUESTED** [review]

File: infra/azure/modules/batch/main.tf (hunk 1 above)
Comment (daniel-goldstein) [reply infra/azure/modules/batch/main.tf:64]: Good catch, I removed the instruction to generate the key.

**Comment by jigold** [comment]
This all seems fine to me, but all of the test jobs failed spectacularly on Azure -- timed out. There must be another place the key is used?

**Review by jigold — APPROVED** [review]

**Comment by danking** [comment]
WIP until I sort the release.

**Comment by daniel-goldstein** [comment]
Turns out you need a key or a password. Closing this for now

**Pull request closed without merging**